### ISSUE TITLE
Add Space and Sizing component to Group block

### ISSUE THREAD

**Issue by georgeolaru** [pixelgrade/nova-blocks#328]
There are few situations when the Group block changes its spacing, particularly when mixing it with the **Color Signal** component. After some discussion with @razwan, we might fix most of the issues if we add integration with the **Space and Sizing** component.

I created a page where we can test various use cases (standalone Group blocks or inside other blocks like Hero or Sidecar block): https://demos.pixelgrade.com/tosca/blocks-group/

Here is a screenshot of how it looks now, highlighting some of the issues: https://user-images.githubusercontent.com/1632775/126153428-d47deae8-5aa1-4490-b043-988ed93931d3.jpeg — as you can see, there are some missing side spacing when the Group block is placed inside the Hero or Sidebar.

There is a sensible case where the Group block with `Color Signal: None` should blend with the background and not have any padding inside. @razwan I will let you dig deeper and decide what is the best approach on this case.

**Comment by madalingorbanescu** [comment]
We've added Space and sizing on the Group block and along with that, we have also added the component on Hero and Slideshow block. (see https://github.com/pixelgrade/nova-blocks/commit/0dc168ed802261e28c313381b2a9b8c0290b3003).

The last part, regarding what `Color Signal: None` should do, @razwan is already tackling that, looking after a solution to apply that signal, relative to the parent.
